server/routes/visit/updateVisitController.test.ts

```diff
@@ -26,7 +26,7 @@ jest.mock('../visitorUtils', () => {
 })
 
 beforeEach(() => {
-  const fakeDate = new Date('2022-01-01')
+  const fakeDate = new Date('2022-01-01T08:00:00')
```

Comment:
Not specifying a time on the fake date can do odd things because it'll default to `00:00:00` and you get into timezone issues.